lib/pf/Switch/Aruba.pm

```diff
@@ -498,7 +498,7 @@ sub radiusDisconnect {
         $attributes_ref = { %$attributes_ref, %$add_attributes_ref };
 
         # Roles are configured and the user should have one
-        if ( defined($role) && (defined($node_info->{'status'}) && isenabled($self->{_RoleMap}) ) ) {
+        if ( defined($role) && isenabled($self->{_RoleMap}) && isenabled($self->{_useCoA}) ) {
```

Comment:
Forget about it :) just another joke out of my bad jokes bucket (which happen to contain all my jokes)